File: templates/algorithm/java4/src/algorithmia/__ALGO__/__ALGO__.java
package algorithmia.__ALGO__;

import com.algorithmia.*;
import com.algorithmia.algo.*;
import com.algorithmia.data.*;
import com.google.gson.*;
import java.util.*;

/**
 * This class defines your algorithm, and its input/output.
 * Algorithms must define at least one apply(...) method.
 *
 * Examples:
 *   public int apply(int[][] array) {...}
 *   public String apply(Map<String,String> object) {...}
 *   public List<Double> apply(double a, double b, double c) {...}
 */
public class __ALGO__ {
    // The input and output of apply() automatically turns into JSON
    public String apply(String input) throws Exception {
        // Your algorithm code goes here
        return "Hello " + input;
    }
}

File: templates/algorithm/python2-2/__ALGO__/__ALGO__.py
import Algorithmia

# This module defines your algorithm, and its input/output.
# You must define an apply function that takes exactly one input.
# The input and output of apply() automatically turns into JSON.
#
# Examples:
#   def apply(array):
#   def apply(str):
#   def apply(x):

def apply(input):
    return 'Hello ' + str(input)
